canonical-web-and-design/ubuntu.com
# Build new /stories page for core

## Done

- Build new page at `/core/stories`
- Add to secondary nav
- NOTE - waiting on correct svgs and design review - padding on some images means alignemnt is slightly off

## QA

- View page at: https://ubuntu-com-9110.demos.haus/core/stories
- Check against [copy doc](https://docs.google.com/document/d/1mXnYnpaCJznDLtYn9Q74aVeEI3iFxVOMzF0pdL7wdu0/edit#heading=h.6hx4umk9oda9)


## Issue / Card

Fixes [#3667](https://github.com/canonical-web-and-design/web-squad/issues/3667)
